package.json

```diff
@@ -20,7 +20,11 @@
   },
   "dependencies": {
     "accessible-autocomplete": "^1.6.2",
-    "sass-mq": "^4.0.2"
+    "node-sass": "^4.11.0",
```

Comment:
Howcome we have installed `node-sass`, `npm` and `start` as dependencies? `dependencies` are for production dependencies and `devDependencies` are for development dependencies (so they will get installed within peoples projects when they `npm install nhsuk-frontend`

If these are required only for BackstopJS to run, can we install these only on the Docker container?